infrastructure/cdn-in-a-box/enroller/enroller.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	log "github.com/apache/trafficcontrol/lib/go-log"
 	tc "github.com/apache/trafficcontrol/lib/go-tc"
-	"github.com/apache/trafficcontrol/traffic_ops/client"
+	"github.com/apache/trafficcontrol/traffic_ops/v4-client"
```

Comment:
I believe this line is causing the build failure:
```
2021-02-04T18:51:49.1234730Z #16 17.29 ./enroller.go:680:29: toSession.CreateServerWithHdr undefined (type *session has no field or method CreateServerWithHdr)
```
The fix would be to link the enroller against the "stable" API client - v3 - or to update references to `CreateServerWithHdr` to simply `CreateServer` (as there is no version in v4 that doesn't use a header).

Reply:
Using `CreateServer()` in 97ea08c368.